docs/index.rst

```diff
@@ -9,17 +9,19 @@ The stellar parameters are age (t), mass (M), metallicity (M), and distance (d).
 The dust extinction parameters are dust column (Av), average grain size (Rv),
 and mixing between type A and B extinction curves (fA).
 
-The full details of the BEAST are provide by
-Gordon et al. (2016, ApJ, 826, 104).
-<http://adsabs.harvard.edu/abs/2016ApJ...826..104G>
+The full details of the BEAST are provided by
+Gordon et al. (2016, ApJ, 826, 104): http://adsabs.harvard.edu/abs/2016ApJ...826..104G
 
-Installation
-============
+Getting started
+===============
 
 .. toctree::
    :maxdepth: 1
 
-   How to install <install.rst>
+   Installation instructions <install.rst>
+     For users <install.rst>
+     For developers <beast_development.rst>
+     For production runs <install.rst>
```

Comment:
The issue is the toctree as you suspected.  If you want sub-bullets, this is created by the sections in a file.  You would need to move all the install instructions into a single file (e.g., install.rst) and then have subsections for each of the 3 sections you want.  Then change the parameter `:maxdepth: 2` (to 2).

Reply:
Yeah, that's what I thought of as a workaround as well. I just hoped there would be a way to fix it without putting everything into a single file. Anyway, I guess this is the only way to fix it. The only remaining problem then will be that if I use maxdepth of 2, also the other titles will have sub-bullets (which are not so useful e.g. for the running an example section).